cupy/random/generator.py

```diff
@@ -223,6 +223,29 @@ def standard_normal(self, size=None, dtype=float):
         """
         return self.normal(size=size, dtype=dtype)
 
+    def tomaxint(self, size=None):
+        """Draws integers between 0 and max integer inclusive.
+
+        Args:
+            size (int or tuple of ints): Output shape.
+
+        Returns:
+            Drawn samples.
+
+        .. seealso::
+            :meth:`numpy.random.RandomState.tomaxint`
+
+        """
+        if size is None:
+            size = ()
+        sample = cupy.empty(size, dtype='l')
+        # cupy.random only uses int32 random generator
+        curand.generate(
+            self._generator, sample.data.ptr, sample.size * 2)
+        # Disable sign bit
+        sample &= 0x7FFFFFFFFFFFFFFF
```

Comment:
Why do not use `sys.maxint`?
`sys.maxint` is different between Linux and Windows.